modules/tinymce/src/core/main/ts/api/Editor.ts

```diff
@@ -449,6 +449,10 @@ class Editor implements EditorObservable {
     return getParam(this, name, defaultVal, type);
   }
 
+  public hasPlugin(key: string): boolean {
+    return Tools.inArray(getParam(this, 'plugins', '', 'string').split(/[ ,]/), key) !== -1;
```

Comment:
Can you please use `Arr.contains` instead, as it'll save the `-1` check.

Additionally, this has dropped something the other  code included which is the `PluginManager.get(name)` check which is important, as the plugin may have failed to load.

- [ ] Use `Arr.contains`
- [ ] Fix regression in functionality (and add a test if possible)

Reply:
As this is often done in the init-calls, before all plugins have loaded, do we still want to use PluginManager.get for this?

Reply:
Yes, it was done in cases previously so that functionality should remain. We could maybe add a flag to the new API to control if that should be checked for edge cases where it's not used maybe?